infrastructure/pcluster-manager.yaml

```diff
@@ -70,6 +99,16 @@ Metadata:
         default: Initial Admin's Email 
       AdminUserPhone:
         default: Initial Admin's Phone Number
+      OIDCProvider:
+        default: OIDC provider
+      OktaDomain:
+        default: Domain
+      OktaServerId:
+        default: Server Id
```

Comment:
issue(non-blocking): "domain" and "Server id" are too vague. Since it's the OktaDomain and the authorizationServerId, I would prepend the work okta to the domain, and maybe even to the authorizationServerId (I'll leave the choice to you) so to make it clear what these refers to.

Reply:
To clarify, the actual parameter is called `OktaServerId`, the `Server Id` is the label visible in the CloudFormation console, in this case I have omitted the "Okta" prefix since these parameters appear in the "Okta" section.